Makefile

```diff
@@ -58,13 +58,13 @@ build-natives:
 
 build-contained:
 	rm -f mender-artifact && \
-	image_id=$$(docker build -f Dockerfile . | awk '/Successfully built/{print $$NF;}') && \
+	image_id=$$(docker build -f Dockerfile -q .) && \
```

Comment:
As the `Makefile` can be used outside of our CI, I think we should still support building without BUILDKIT. Can we check for it somehow in this command to then conditionally use `awk`?

Reply:
=> https://github.com/mendersoftware/mender-artifact/pull/414#issuecomment-1220657061 - this works for both...

Reply:
Sorry :facepalm: